src/cloud-api-adaptor/podvm/files/etc/systemd/system/attestation-agent.service

```diff
@@ -1,11 +1,15 @@
 [Unit]
 Description=Attestation Agent TTRPC API Server
-After=network.target process-user-data.service
+After=network.target cloud-final.service process-user-data.service
```

Comment:
to fix issue at https://github.com/confidential-containers/cloud-api-adaptor/pull/1868#issuecomment-2188451116
`cloud-final.service` -> `cloud-init.service`

Reply:
Thanks @liudalibj let me try whether the toml file placed rightly with this change.

Reply:
looks not good, cloud-init.service can not guarantee the cloud-init files placed correctly,  cloud-final.service can. This might be the difference between ubuntu and fedora.

Reply:
Let me have a try to overwrite it in fedora and keep it in ubuntu.

Reply:
Tried again, I misused the image in my env, looks `cloud-init.service` is a good approach.

Reply:
Double checked, we can use `cloud-init.service` instead cloud-final.service. the IBM SE attestation works well on fedora image, the sample tee works for ubuntu image in e2e.